base/docs/helpdb/Base.jl

```diff
@@ -351,79 +351,6 @@ See [`RoundingMode`](:obj:`RoundingMode`) for available rounding modes.
 Float32
 
 """
-    Mmap.mmap(io::Union{IOStream,AbstractString,Mmap.AnonymousMmap}[, type::Type{Array{T,N}}, dims, offset]; grow::Bool=true, shared::Bool=true)
```

Comment:
Because I wanted to match the method the docstring is attached to. `IO` is a more general type than that `Union`, right? If the core implementation of `mmap` only actually takes those three types, we should change the method signature, shouldn't we?